.github/workflows/pr-license-check.yaml

```diff
@@ -4,7 +4,11 @@ on:
   push:
     branches:
     - main
+    paths:
+    - 'cnap/**/requirements.txt'
   pull_request:
+    paths:
+    - 'cnap/**/requirements.txt'
```

Comment:
Only check requirements.txt license?
Shouldn't we check all source code?

Reply:
For license check, we use [liccheck](https://pypi.org/project/liccheck/) to check python packages listed in `requirement.txt` file, so we don't need to check all source code in license check.